CONTRIBUTING.md

```diff
@@ -60,12 +61,6 @@ appropriate to start the first line of a commit with '(doc)' instead of
 a ticket number. 
 
 ````
-    (doc) Add documentation commit example to CONTRIBUTING
```

Comment:
Why were the first lines of the example commit messages removed?  I think these either need to be complete examples (as they previously were) or they need to be removed altogether and replaced with a link to documentation on writing a good commit message.

Reply:
It seemed like boilerplate text which didn't make sense to me. If it does make sense to keep, I'll add it back. L31-L38 and L63-L68 are the blocks I'm referencing.

Reply:
It is a little confusing, but it's supposed to be an example commit message (for a commit about adding the commit message example, which is probably why it gets confusing).  Maybe using a more concrete example would be better.

Reply:
How is the latest update?

Reply:
perfect